chrome/content/zotero/xpcom/date.js

```diff
@@ -369,7 +371,7 @@ Zotero.Date = new function(){
 		}
 		
 		// MONTH
-		if(!date.month) {
+		if(date.month === undefined) {
```

Comment:
It fixes https://github.com/zotero/zotero/pull/820/files#diff-416fa9a7f56dfe0b3a5afd2238dfb317L1351 because of https://github.com/zotero/zotero/pull/820/files#diff-a098326f7cdaa37adf1398d92b384be5R1615 and the code above guarantees that `date.month` is `undefined`